hugo/data/you_draw_it_koeln_paris_data.json

```diff
@@ -1 +1,39 @@
-{"values": [{"year": 1990, "value": 12130}, {"year": 2008, "value": 10250}, {"year": 2015, "value": 10000}, {"year": 2020.0, "value": 10000.0}, {"year": 2021.0, "value": 8912.50357837789}, {"year": 2022.0, "value": 7825.00715675578}, {"year": 2023.0, "value": 6737.51073513367}, {"year": 2024.0, "value": 5650.0143135115595}, {"year": 2025.0, "value": 4562.517891889449}, {"year": 2026.0, "value": 3475.0214702673393}, {"year": 2027.0, "value": 2387.5250486452296}, {"year": 2028.0, "value": 1300.028627023119}, {"year": 2029.195432556076, "value": 0.0}, {"year": 2029, "value": 0}, {"year": 2030, "value": 0}, {"year": 2031, "value": 0}, {"year": 2032, "value": 0}, {"year": 2033, "value": 0}, {"year": 2034, "value": 0}, {"year": 2035, "value": 0}, {"year": 2036, "value": 0}, {"year": 2037, "value": 0}, {"year": 2038, "value": 0}, {"year": 2039, "value": 0}, {"year": 2040, "value": 0}, {"year": 2041, "value": 0}, {"year": 2042, "value": 0}, {"year": 2043, "value": 0}, {"year": 2044, "value": 0}, {"year": 2045, "value": 0}, {"year": 2046, "value": 0}, {"year": 2047, "value": 0}, {"year": 2048, "value": 0}, {"year": 2049, "value": 0}, {"year": 2050, "value": 0}]}
```

Comment:
Zur Info: Diese Datei wird automatisch erzeugt, und [ab jetzt](https://github.com/codeformuenster/klimawatch/commit/3ff2f288b37da4df256dedc1d6d36941e1a30306) auch schöner formattiert. Noch nicht implementiert ist, dass der you-draw-it-chart diese Datei auch einliest (habe ich bisher manuell in die andere .json kopiert)

Reply:
alles klar, lasse die dann in ruhe so weit :D